x11-libs/startup-notification/startup-notification-0.12-r2.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit xorg-3
```

Comment:
empty line after `EAPI` please.

Reply:
Oh, I was doing this for modern c porting and hence did the revision bump

Reply:
We need revbumps for modern C fixes usually, but this one is a test-only fix, so it doesn't change what is installed -> no revision.